File: app/src/main/java/com/zl/weilu/androidut/dagger/ui/BuildersModule.java
package com.zl.weilu.androidut.dagger.ui;

import com.zl.weilu.androidut.dagger.base.scope.ActivityScope;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

/**
 * 作者：weilu on 2018/2/5 09:46
 */

@Module
public abstract class BuildersModule {

    @ActivityScope
    @ContributesAndroidInjector
    abstract LoginDaggerActivity loginDaggerActivityInjector();

}

File: app/src/main/java/com/zl/weilu/androidut/dagger/ui/BuildersModule.kt
package com.zl.weilu.androidut.dagger.ui

import com.zl.weilu.androidut.dagger.base.scope.ActivityScope

import dagger.Module
import dagger.android.ContributesAndroidInjector

/**
 * 作者：weilu on 2018/2/5 09:46
 */

@Module
abstract class BuildersModule {

    @ActivityScope
    @ContributesAndroidInjector
    internal abstract fun loginDaggerActivityInjector(): LoginDaggerActivity

}
